Documentation/conf.py

```diff
@@ -44,7 +44,18 @@
               'sphinx_tabs.tabs',
               'sphinxcontrib.googleanalytics',
               'sphinxcontrib.spelling',
-              'versionwarning.extension']
+              'versionwarning.extension',
+              "sphinxext.rediraffe",
+]
+
+rediraffe_redirects = 'redirects.txt'
+# rediraffe_branch is the base for which rediraffe compares the current HEAD to
+# to detect which Documentation pages have moved or been deleted so it can
+# check for missing redirects and automatically generate new redirect files.
+# The value specified is the commit before we branched v1.16 found using:
+# `git merge-base v1.16 main`
+rediraffe_branch = '5614531067a83e20d24bccc7b12b314330d043c3'
```

Comment:
Looks good, but one note: we also have many redirects in place in the RTD interface. Ideally, it would be nice to move them to rediraffe at some point in the future to have all redirects managed at a single location. Not sure how to reliably reproduce the current RTD list on this side, though. We'd have to update the commit ref above, and the list of redirects, but it will probably not be trivial to regenerate the same :thinking: